org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/thingTranslator.js

```diff
@@ -79,7 +79,7 @@ function providerSchemaToPlatformSchema(providerSchema, expand) {
         entities: [
             {
                 n: providerSchema['name'],
-                ivc: "core.1.1.0",
+                icv: "core.1.1.0",
```

Comment:
>icv [](start = 16, length = 3)

huh -- so how was this working to begin with ? Is this because we never tested this and are catching it now because of your changes or because of some recent prod change?